content/docs/1.9.0/important-notes/_index.md

```diff
@@ -132,7 +149,10 @@ For more information, see [#10550](https://github.com/longhorn/longhorn/issues/1
 
 ### Longhorn System Upgrade
 
-Longhorn currently does not support live upgrading of V2 volumes. Ensure that all V2 volumes are detached before initiating the upgrade process.
+Longhorn currently does not support live upgrading of V2 volumes or V2 backing images. Before starting the upgrade process:
+
+- Detach all V2 volumes from their workloads.
+- Backup any volumes created from V2 backing images to avoid data loss.
```

Comment:
The change is redundant because you have mentioned the backing image issue before.